backend/osmosewebsite/urls.py

```diff
@@ -1,15 +1,18 @@
 """OSmOSE Website API Routing"""
 from rest_framework import routers
+from backend.osmosewebsite.views.trap import TrapViewSet
```

Comment:
Should be included in views import below

Reply:
Nothing has changed here @ussefhajji Can you move the import of TrapViewSet in the imports below?